db/migrate/20230328152324_add_users_ref_to_likes.rb

```diff
@@ -1,5 +1,5 @@
 class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
   def change
-    add_reference :likes, :author, foreign_key: true
+    add_reference :likes, :author, foreign_key: { to_table: :users }
```

Comment:
- [ ] You did a good job working on the previous reviewer's requirements 🚀  However, I'm still getting the same errors as before, this happens because the Rails conventions that by default look for `post_id` instead of `posts_id` as the name of your model is `Post`. 🤔 

I managed to solve this problem by following the next steps:

1. Look for `add_users_ref_to_posts` migration and change `foreign_key: true` to `{ to_table: :users }`.
2. Look for `add_users_ref_to_comments` migration and change `foreign_key: true` to `{ to_table: :users }`.
3. Run the following command to reset the database and run all migrations again, updating the schema:
```sh
rails db:drop db:create db:migrate
```
4. Run the migrations also in your test env:
```sh
rails db:migrate RAILS_ENV=test
```
5. You should now be good and tests are going to fail because of the changes you made by adding an `s` to `posts_id`.
6. Change every `posts_id` to `post_id` and tests should pass, and the project can be approved!

Please consider making this fix, Cheers and happy coding! 🤓 🔧